index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- stylesheet -->
+		<link rel="stylesheet" href="style.css" />
+		<!-- fontawesome -->
+			<script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+		<!-- google font -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700&family=Poppins:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<title>Portfolio Website</title>
+	</head>
+	<body>
+		<div class="intro-container">
+			<header>
+				<h1>Victor</h1>
```

Comment:
- [ ]  Nice job adding a logo to your page 👍. Please consider wrapping the logo in an `a` tag as the logo should be a link that leads to the homepage and the `h1` tag should be used for the main title or heading in your project. See my suggestion below.

```suggestion
			<header>
				<a href="">Victor</a>
```